My last pass over the intro (i.e. before the datalad overview) (#88)
datalad/datalad-paper-joss: paper.md

In order to maximize its utility and target audience, DataLad is available for all major operating systems, and can be integrated into established workflows and environments with minimal friction.

# Statement of Need

Code, data and computing environments are core components of scientific projects.
While the collaborative development and use of research software and code is streamlined with established procedures and infrastructures, such as software distributions, distributed version control systems, and social coding portals like GitHub, other components of scientific projects are not as transparently managed or unobstructed accessible.
Data consumption is complicated by disconnected data portals that require a large variety of different data access and authentication methods.
(Re)used data are not as precisely identified as it is standard for code in software development, because data versioning is rarely or only coarsely practiced.
Scientific computation is not reproducible enough, because data provenance, the information of how a digital file came to be, is often incomplete and rarely automatically captured.

> Last but not least, in the absence of standardized data *packages*, there is no uniform way to declare actionable data dependencies and derivative relationships between inputs and outputs of a computation.

Reviewer: Entire Statement of Need reads much better!

To achieve modularity without impeding usability, DataLad simplifies working with the resulting hierarchies of Git repositories via recursive operations across dataset boundaries.
With this, DataLad provides a "mono-repo"-like user experience in datasets with arbitrarily deep nesting, and makes it trivial to operate on individual files deep in the hierarchy or entire trees of datasets.
A testament of this is [datasets.datalad.org](http://datasets.datalad.org), created as the project's initial goal to provide a data distribution with unified access to already available public data archives in neuroscience, such as [crcns.org](http://crcns.org) and [openfmri.org](http://openfmri.org).
It is curated by the DataLad team, and provides, at the time of publication, streamlined access to over 250 TBs of data across a wide range of projects and archives in a fully modularized way.

**Re-executable annotation of changes.**
Digital provenance is crucial for the trustworthiness and reproducibility of a research result, and contributes to the reusability aspect of the FAIR principles [@FAIR2016].

> Git captures provenance with annotations of changes that are typically represented by a patch (an exact difference between two versions that could be applied to another version of the text file), and a commit message (a freeform, human-readable text description of the introduced changes).

Reviewer: Not really about this PR, but this sentence still is confusing in its structure. How about this?

```suggestion
Git captures provenance with annotations of changes, where changes are typically represented by a patch (an exact difference between two versions that could be applied to another version of the text file), and annotations by a commit message (a freeform, human-readable text description of the introduced changes).
```

Author: The sentence is (without parenthesis):

> Git captures provenance with annotations of changes that are typically represented by a patch, and a commit message.

I think your proposal is more correct, but arguably less attractive ;-)

What about

```suggestion
Git captures provenance by annotating a patch (an exact difference between two versions that could be applied to another version of the text file), with a commit message (a freeform, human-readable text description of the introduced changes).
```

Reviewer: Yes, better. :)

**Targeted interfaces and interoperability adapters.**
Interoperability with scientific or commercial computing and storage services allows researchers to integrate data management routines into their established workflows with minimal friction.
Git can already interact with other local or remote repositories via a set of standard (ssh, http) or custom (Git) network transport protocols.
DataLad implements support for additional services that require custom protocols, such as the Open Science Framework (OSF) [@datalad-osf:zenodo].
Git-annex readily provides access to a wide range of external data storage resources via a large set of protocols.

> DataLad build on this support and adds, for example, more fine-grained access (e.g.

Reviewer: ```suggestion
DataLad builds on this support and adds, for example, more fine-grained access (e.g.
```
